janusgraph-dist/janusgraph-dist-hadoop-2/Dockerfile

```diff
@@ -6,6 +6,15 @@ ADD ${server_zip} /var
 RUN apt-get update -y && apt-get install -y zip && \
     server_base=`basename ${server_zip} .zip` && \
     unzip -q /var/${server_base}.zip -d /var && \
-    ln -s /var/${server_base} /var/janusgraph
+    rm /var/${server_base}.zip && \
+    ln -s /var/${server_base} /var/janusgraph && \
+    groupadd -g 999 janusgraph && \
```

Comment:
Not that important, but is there a reason why you specified the group and user ids explicitly?

Reply:
You mean the 999? Well i remember it being problematic when mounting the filesystem from another system. If janusgraph user has a different userid on a different system, then you don't have the exact same privs for files created by the "original" janusgraph. IMHO.

Reply:
OK, thanks for clarifying.